argocd/metadata.csv

```diff
@@ -0,0 +1,152 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+argocd.api_server.go.gc.duration.seconds,gauge,,second,,A summary of the pause duration of garbage collection cycles in the API Server,0,argocd,api_server go gc duration seconds,
+argocd.api_server.go.gc.duration.seconds.count,count,,second,,The summary count of garbage collection cycles in the Repo Server API Server,0,argocd,api_server go gc duration seconds count,
+argocd.api_server.go.gc.duration.seconds.sum,gauge,,second,,The sum of the pause duration of garbage collection cycles in the API Server,0,argocd,api_server go gc duration seconds sum,
+argocd.api_server.go.goroutines,gauge,,,,The number of goroutines that currently exist in the API Server,0,argocd,api_server go goroutines,
+argocd.api_server.go.memstats.alloc_bytes,gauge,,byte,,The number of bytes allocated and still in use in the API Server,0,argocd,api_server go memstats alloc_bytes,
+argocd.api_server.go.memstats.buck_hash.sys_bytes,gauge,,byte,,The number of bytes used by the profiling bucket hash table in the API Server,0,argocd,api_server go memstats buck_hash sys_bytes,
+argocd.api_server.go.memstats.frees,count,,,,The notal number of frees in the API Server,0,argocd,api_server go memstats frees,
+argocd.api_server.go.memstats.gc.cpu_fraction,gauge,,fraction,,The fraction of this program's available CPU time used by the GC since the program started in the API Server,0,argocd,api_server go memstats gc cpu_fraction,
+argocd.api_server.go.memstats.gc.sys_bytes,gauge,,byte,,The number of bytes used for garbage collection system metadata in the API Server,0,argocd,api_server go memstats gc sys_bytes,
+argocd.api_server.go.memstats.heap.alloc_bytes,gauge,,byte,,The number of heap bytes allocated and still in use in the API Server,0,argocd,api_server go memstats heap alloc_bytes,
+argocd.api_server.go.memstats.heap.idle_bytes,gauge,,byte,,The number of heap bytes waiting to be used in the API Server,0,argocd,api_server go memstats heap idle_bytes,
+argocd.api_server.go.memstats.heap.inuse_bytes,gauge,,byte,,The number of heap bytes that are in use in the API Server,0,argocd,api_server go memstats heap inuse_bytes,
+argocd.api_server.go.memstats.heap.objects,gauge,,object,,The number of allocated objects in the API Server,0,argocd,api_server go memstats heap objects,
+argocd.api_server.go.memstats.heap.released_bytes,gauge,,byte,,The number of heap bytes released to OS in the API Server,0,argocd,api_server go memstats heap released_bytes,
+argocd.api_server.go.memstats.heap.sys_bytes,gauge,,byte,,The number of heap bytes obtained from system in the API Server,0,argocd,api_server go memstats heap sys_bytes,
+argocd.api_server.go.memstats.last_gc_time.seconds,gauge,,second,,The number of seconds since 1970 of last garbage collection in the API Server,0,argocd,api_server go memstats last_gc_time second,
```

Comment:
```suggestion
argocd.api_server.go.memstats.last_gc_time.seconds,gauge,,second,,The number of seconds since Unix epoch of last garbage collection in the API Server,0,argocd,api_server go memstats last_gc_time second,
```